app/models/actor.rb

```diff
@@ -80,6 +73,18 @@ class Actor < ApplicationRecord
 
   after_destroy :update_index_async
 
+  def created_work_versions
+    authorships.where(resource_type: 'WorkVersion')
```

Comment:
I think this is confusing, because unless I am reading this wrong, this method returns back an array of Author methods that belong to Work Versions, not an array of WorkVersions themselves.

I think this should probably be an association instead. 
`has_many :work_versions, through: :authorships, -> { where resource_type: 'WorkVersion' }`

Reply:
Thanks, I didn't know how it would work with the polymorphism. I'm honestly not sure if these associations are even useful. There can't be an inverse on `WorkVersion` because it would exclude those authors that don't have actor references. So it seems it's just a convenient kind of query. Or, am I forgetting something?